src/ikpy/URDF_utils.py

```diff
@@ -139,11 +139,16 @@ def get_urdf_parameters(urdf_file, base_elements=["base_link"], last_link_vector
         translation = joint.find("origin").attrib["xyz"].split()
         orientation = joint.find("origin").attrib["rpy"].split()
         rotation = joint.find("axis").attrib['xyz'].split()
+        parent = None
```

Comment:
Wow, sorry for this one. I added it because I though I would use it for the blender importer, but finally I don't. Actually I only use dictionaries in the blender importer, so that the parent is just a string in the field "parent".
It is better to remove it, I guess ?